Makefile

```diff
@@ -28,6 +29,9 @@ VENV ?= $(OUTPUT)/venv
 export OPERATOR_SDK_VERSION ?= v0.17.2
 export OPERATOR_SDK ?= build/_tools/operator-sdk-$(OPERATOR_SDK_VERSION)
 
+OPERATOR_REGISTRY_VERSION ?= v1.13.3
+OPERATOR_REGISTRY_REPO_URL ?= https://github.com/operator-framework/operator-registry
+
```

Comment:
Let's remove this per @guymguym suggestion.

Reply:
Addressed in 8857002